### PR TITLE
Enh: Rename project from Shinken to Alignak. Keep headers

### PR DESCRIPTION
**Pull request by Seb-Solon**
Similar to #5

### CODE REVIEW

File: alignak/arbiterlink.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Copyright (C) 2015-2015: Alignak team, see AUTHORS.txt file for contributors
```
Comment (Seb-Solon) [inline alignak/arbiterlink.py]: missing #!/usr/bin/env python

File: alignak/brokerlink.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Copyright (C) 2015-2015: Alignak team, see AUTHORS.txt file for contributors
```
Comment (Seb-Solon) [inline alignak/brokerlink.py]: #!/usr/bin/env python

File: alignak/daemons/__init__.py [hunk 3]
```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/python
```
Comment (Seb-Solon) [inline alignak/daemons/__init__.py]: Wrong #!/usr/bin/env python

File: alignak/daemons/pollerdaemon.py [hunk 4]
```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/python
```
Comment (Seb-Solon) [inline alignak/daemons/pollerdaemon.py]: Wrong

File: alignak/discovery/__init__.py [hunk 5]
```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/python
```
Comment (Seb-Solon) [inline alignak/discovery/__init__.py]: Wrong

File: alignak/misc/__init__.py [hunk 6]
```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/python
```
Comment (Seb-Solon) [inline alignak/misc/__init__.py]: Wrong

File: alignak/webui/__init__.py [hunk 7]
```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/python
```
Comment (Seb-Solon) [inline alignak/webui/__init__.py]: Wrong

File: alignak/arbiterlink.py (hunk 1 above)
Comment (gst) [inline alignak/arbiterlink.py]: nope ;)

the shebang line should only be added to the file aimed to be executed, that is to the daemons only (or any other tool).

File: alignak/clients/__init__.py [hunk 8]
```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/python
```
Comment (Seb-Solon) [inline alignak/clients/__init__.py]: Wrong

**Comment by gst** [comment]
"nice" job ;)

**Comment by gst** [comment]
that's good enough for me; tests passes so it's ok. let's merge and move forward!

**Pull request merged**